18 - Apple HomePage/html/index.html

```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
delete the folder "HTML" and put index file directly in "apple homepage" folder.